### PR TITLE
login-error-messages

### PR DESCRIPTION
**Pull request by Negi-Anurag**

### CODE REVIEW

File: server/routes/login.js [hunk 1]
```diff
@@ -115,10 +115,10 @@ router.post('/', async (req, res) => {
         return res.status(409).send({
           message: 'Authentication failed.',
         });
-      }else if(!establishmentUser.isActive && establishmentUser.status === 'Pending'){
+      } else if(!establishmentUser.isActive && establishmentUser.status === 'PENDING'){
         //check for Pending status, if Pending then return with 403 error
         console.error(`POST .../login failed: User status is pending`);
-        return res.status(403).send({
+        return res.status(405).send({
```
Comment (aaron-russell) [inline server/routes/login.js]: Is there any unit tests around login?

Comment (Negi-Anurag) [reply server/routes/login.js]: I am not sure on this. I worked on FE part, due angular issue with 403 I have changed it to 405.

Comment (aaron-russell) [reply server/routes/login.js]: Just wanted to make sure a unit test wouldn't fail if this is changed

Comment (Negi-Anurag) [reply server/routes/login.js]: I am merging it as of now it will stay in feature branch only.

**Pull request merged**